manifests/init.pp

```diff
@@ -133,6 +134,13 @@
   validate_string($computers_ou)
   validate_string($nismaps_ou)
 
+  if is_string($domain_change) {
+    $domain_change_real = str2bool($domain_change)
+  } else {
+    $domain_change_real = $domain_change
+  }
+  validate_bool($domain_change_real)
```

Comment:
Hello,

I can just add the domain_change parameter to this part of the code and it would be tested.
https://github.com/Ericsson/puppet-module-vas/blob/development/spec/classes/init_spec.rb#L1314-L1342